test/command/suite/sharding/logical_select/columns/stage/filtered/window_function/window_count/ascending.expected

```diff
@@ -0,0 +1,130 @@
+plugin_register sharding
+[[0,0.0,0.0],true]
+table_create Logs_20170415 TABLE_NO_KEY
+[[0,0.0,0.0],true]
+column_create Logs_20170415 timestamp COLUMN_SCALAR Time
+[[0,0.0,0.0],true]
+column_create Logs_20170415 price COLUMN_SCALAR UInt32
+[[0,0.0,0.0],true]
+column_create Logs_20170415 n_likes COLUMN_SCALAR UInt32
+[[0,0.0,0.0],true]
+table_create Logs_20170416 TABLE_NO_KEY
+[[0,0.0,0.0],true]
+column_create Logs_20170416 timestamp COLUMN_SCALAR Time
+[[0,0.0,0.0],true]
+column_create Logs_20170416 price COLUMN_SCALAR UInt32
+[[0,0.0,0.0],true]
+column_create Logs_20170416 n_likes COLUMN_SCALAR UInt32
+[[0,0.0,0.0],true]
+load --table Logs_20170415
+[
+{"timestamp": "2017/04/15 00:00:00", "n_likes": 2, "price": 100},
+{"timestamp": "2017/04/15 01:00:00", "n_likes": 1, "price": 100},
+{"timestamp": "2017/04/15 01:00:00", "n_likes": 2, "price": 200}
+]
+[[0,0.0,0.0],3]
+load --table Logs_20170416
+[
+{"timestamp": "2017/04/16 10:00:00", "n_likes": 1, "price": 200},
+{"timestamp": "2017/04/16 11:00:00", "n_likes": 2, "price": 300},
+{"timestamp": "2017/04/16 11:00:00", "n_likes": 1, "price": 300}
+]
+[[0,0.0,0.0],3]
+logical_select Logs   --shard_key timestamp   --filter 'price > 100'   --columns[count].stage filtered   --columns[count].type UInt32   --columns[count].flags COLUMN_SCALAR   --columns[count].value 'window_count()'   --columns[count].window.sort_keys _id   --output_columns _id,n_likes,count
+[
+  [
+    0,
+    0.0,
+    0.0
+  ],
+  [
+    [
+      [
+        4
+      ],
+      [
+        [
+          "_id",
+          "UInt32"
+        ],
+        [
+          "n_likes",
+          "UInt32"
+        ],
+        [
+          "count",
+          "UInt32"
+        ]
+      ],
+      [
+        3,
+        2,
+        1
+      ],
+      [
+        1,
+        1,
+        2
+      ],
+      [
+        2,
+        2,
+        3
+      ],
+      [
+        3,
+        1,
+        4
+      ]
+    ]
+  ]
+]
+logical_select Logs   --shard_key timestamp   --filter 'price > 100'   --columns[count].stage filtered   --columns[count].type UInt32   --columns[count].flags COLUMN_SCALAR   --columns[count].value 'window_count()'   --columns[count].window.sort_keys n_likes   --output_columns _id,n_likes,count
+[
+  [
+    0,
+    0.0,
+    0.0
+  ],
+  [
+    [
+      [
+        4
+      ],
+      [
+        [
+          "_id",
+          "UInt32"
+        ],
+        [
+          "n_likes",
+          "UInt32"
+        ],
+        [
+          "count",
+          "UInt32"
+        ]
+      ],
+      [
```

Comment:
n_likes key works as expected?

Reply:
Hmm, I'm quite misunderstand window function behavior.

Reply:
@komainu8 could you directly ask kou to review, please? I'm no confidence to review correctly.

Reply:
@kenhys OK.Thank you for your review.